dakara_server/library/views.py

```diff
@@ -24,8 +24,10 @@ class LibraryPagination(PageNumberPagination):
     """
     def get_paginated_response(self, data):
         return Response({
-                'current': self.page.number,
-                'last': self.page.paginator.num_pages,
+                'pagination': {
+                    'current': self.page.number,
+                    'last': self.page.paginator.num_pages,
+                    },
                 'count': self.page.paginator.count,
```

Comment:
Isn't `count` also part of pagination, it's here only because of pagination.

If there were not pagination, count wouldn't be needed, the size of `results` would be enough